python/pysnaptest/snapshot.py

```diff
@@ -1,14 +1,17 @@
+from __future__ import annotations
```

Comment:
This and the `if TYPE_CHECKING` below are to avoid python freaking out about pl and pd not being defined when loading this module.